core/server/src/main/java/alluxio/cli/Format.java

```diff
@@ -51,7 +54,7 @@ private static boolean formatFolder(String name, String folder, Configuration co
   }
 
   /**
-   * Formats the Alluxio file system via {@code java -cp %s alluxio.Format <MASTER/WORKER>}.
+   * Formats the Alluxio file system via {@code java -cp %s alluxio.cli.Format <MASTER/WORKER>}.
```

Comment:
if we inline the USAGE message here, we can simplify this to `Format the Alluxio system.`